### PR TITLE
Enable etcd auto compaction in Milvus's Ubuntu package

### LINKED ISSUE #16511: [Bug]: etcd auto compaction not enabled in milvus's Ubuntu package
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: Version: 2.0.2-1 from APT-Sources: http://ppa.launchpad.net/milvusdb/milvus/ubuntu bionic/main amd64 Packages
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus==2.0.1
- OS(Ubuntu or CentOS): Ubuntu 18.04.6 LTS
- CPU/Memory: AWS instance type c5a.4xlarge. AMD EPYC 7002 2nd generation / 32 GB 
- GPU: none
- Others:
```

### Current Behavior

When I insert approximately 10 million vectors over the course of 3 days, Milvus crashes with the following log statement:
```
Apr 14 13:39:08 ip-10-0-84-244 milvus[18584]: panic: etcdserver: mvcc: database space exceeded
```

### Expected Behavior

When I insert approximately 10 million vectors over the course of 3 days, Milvus shouldn't crash.

### Steps To Reproduce

```markdown
1. Install Milvus standalone on Ubuntu 18.04
2. Write and execute a Python program that inserts 10 million vectors over the course of 3 days.
3. Milvus will crash (see above).
```

### Anything else?

The problem seems to be that etcd's `quota-size-bytes` (default=2GB) was exceeded and etcd's auto-compaction was not enabled.

The issue has been reported before (#5519, #6753) and as a fix the Milvus team enabled etcd's auto compaction for Docker and Helm (https://github.com/milvus-io/milvus/issues/5519#issuecomment-926374617). 

**However, it is not enabled in the Milvus standalone Ubuntu package.**

On startup milvus-etcd logs:
```
Apr 16 16:56:54 ... milvus-etcd[3742]: {... 
"auto-compaction-mode":"periodic",
"auto-compaction-retention":"0s",
"auto-compaction-interval":"0s",
...}
```

# Workaround
1. change  `ExecStart` in ` /lib/systemd/system/milvus-etcd.service` to:
`ExecStart=/usr/bin/milvus-etcd --data-dir /var/lib/milvus/etcd-data --auto-compaction-retention '1h' --auto-compaction-mode 'periodic'`
2. `sudo systemctl milvus stop`
3. `sudo systemctl milvus-etcd stop`
4. `sudo systemctl daemon-reload`
5. `sudo systemctl milvus start`

Now milvus-etcd logs:
```
Apr 16 19:04:33 ... milvus-etcd[28307]: {... 
"auto-compaction-mode":"periodic",
"auto-compaction-retention":"1h0m0s",
"auto-compaction-interval":"1h0m0s", 
...}
```

### PR DESCRIPTION
**Pull request by schuberttobias**
Issue: #16511

@smeikalr: FYI, I've set up this PR based on our own fix.

### CODE REVIEW

File: build/rpm/services/milvus-etcd.service [hunk 1]
```diff
@@ -27,4 +27,4 @@ StandardError=inherit
 Restart=always
 
 # Start main service
-ExecStart=/usr/bin/milvus-etcd --data-dir /etc/milvus/etcd-data
```
Comment (schuberttobias) [inline build/rpm/services/milvus-etcd.service:30]: @LoveEachDay: What is the correct value for `--data-dir`? 

In the `master` branch it is `--data-dir /etc/milvus/etcd-data`.
But when I `apt-get install milvus==2.0.2-1` the content of `/etc/systemd/system/milvus-etcd.service` is different: ` --data-dir /var/lib/milvus/etcd-data`. 

image


`/var/lib/milvus/etcd-data/` exists and holds files, `/etc/milvus/etcd-data` does not exist.

**Comment by schuberttobias** [comment]
I think I've applied all changes as requested and explained by your CI. 

Do you have a branch name convention? I did not find one and emulated [@xiaofan-luan 's style](https://github.com/xiaofan-luan/milvus/branches/all)

File: build/rpm/services/milvus-etcd.service (hunk 1 above)
Comment (LoveEachDay) [reply build/rpm/services/milvus-etcd.service:30]: We change the default data directory for milvus 2.0.2.
/cc @Bennu-Li

Comment (LoveEachDay) [reply build/rpm/services/milvus-etcd.service:30]: For ubuntu the default data directory for etcd is `/var/lib/milvus/ectd-data`, and for centos related the default directory for etcd is `/etc/milvus/etcd-data` .

And for now we don't put the service config for ubuntu into milvus repo.

**Comment by LoveEachDay** [comment]
@schuberttobias We've update the data directory for etcd. Could you rebase your code?

**Comment by schuberttobias** [comment]
@LoveEachDay Yes, I rebased onto 1cabaa16a140e8eaaebd785993244f1a19476dd3.

**Pull request closed without merging**